grammars/go.cson

```diff
@@ -85,7 +85,7 @@
   }
   {
     'name': 'storage.type.go'
-    'match': '\\b(chan|map|bool|string|error|int|int8|int16|int32|int64|rune|byte|uint|uint8|uint16|uint32|uint64|uintptr|float32|float64|complex64|complex128)\\b'
+    'match': '\\b(bool|string|error|int|int8|int16|int32|int64|rune|byte|uint|uint8|uint16|uint32|uint64|uintptr|float32|float64|complex64|complex128)\\b'
```

Comment:
It's a bit of a silly example, of course. It's very unlikely to use all type related keywords together like that.

The difference with the new scopes on a more likely example is more appealing:
<img width="185" alt="screen shot 2015-07-24 at 13 08 45" src="https://cloud.githubusercontent.com/assets/703602/8873276/34bee28e-3205-11e5-9d8d-c641bf67afa3.png">
and
<img width="186" alt="screen shot 2015-07-24 at 13 09 09" src="https://cloud.githubusercontent.com/assets/703602/8873275/34bed280-3205-11e5-935d-b5337c5411f3.png">
Builtin functions, keywords, primitive types and other types, are all highlighted differently.